float64 lat
float64 lon
float32 z
float32 yaw
float32 speed
string frame_id
bool auto_arm
---
bool success
string message
